### PR TITLE
chore(macros/HTTPSidebar): remove Cookie Security link

### LINKED ISSUE #8946: The link of "Cookie security" in the sidebar does not exist
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/HTTP/Cookies

### What specific section or headline is this issue about?

_No response_

### What information was incorrect, unhelpful, or incomplete?

The link of "Cookie security" in the sidebar is https://developer.mozilla.org/en-US/docs/Web/HTTP/Cookies#Secure_and_HttpOnly_cookies, while this link doesn't exist in the "Cookies" page

### What did you expect to see?

The link of "Cookie security" in the sidebar may be https://developer.mozilla.org/en-US/docs/Web/HTTP/Cookies#security

### Do you have any supporting links, references, or citations?

_No response_

### Do you have anything more you want to share?

_No response_

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/http/cookies`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/HTTP/Cookies
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/http/cookies/index.md
* Last commit: https://github.com/mdn/content/commit/0880a90f3811475d78bc4b2c344eb4146f25f66c
* Document last modified: 2023-04-10T19:47:15.000Z

</details>

### PR DESCRIPTION
**Pull request by jhongyu**
## Summary

Fixes https://github.com/mdn/yari/issues/8946

### Problem

The link of "Cookie security" in the HTTP security sidebar is `https://developer.mozilla.org/en-US/docs/Web/HTTP/Cookies#Secure_and_HttpOnly_cookies`, while this link doesn't exist in the "Cookies" page

### Solution

Update the link of "Cookie security" in the HTTP security sidebar to `https://developer.mozilla.org/en-US/docs/Web/HTTP/Cookies#security`

---

### Before

https://github.com/mdn/yari/assets/48473551/74367a8a-f094-4608-ba61-b07890fa21e4

### After

https://github.com/mdn/yari/assets/48473551/11dace53-b20b-4d7a-8cf8-a88c2c432f08

---

### CODE REVIEW

File: kumascript/macros/HTTPSidebar.ejs [hunk 1]
```diff
@@ -321,7 +321,7 @@ var text = mdn.localStringMap({
             <ol>
                 <li><a href="/<%=locale%>/docs/Web/HTTP/CSP"><%=text['CSP']%></a></li>
                 <li><a href="/<%=locale%>/docs/Web/HTTP/Headers/Strict-Transport-Security">HTTP Strict Transport Security (HSTS)</a></li>
-                <li><a href="/<%=locale%>/docs/Web/HTTP/Cookies#Secure_and_HttpOnly_cookies">Cookie security</a></li>
+                <li><a href="/<%=locale%>/docs/Web/HTTP/Cookies#security">Cookie security</a></li>
```
Comment (yin1999) [inline kumascript/macros/HTTPSidebar.ejs:324]: The URI fragment is "locale-aware", for example, this should be `"#安全"` for zh-CN, `"#安全性"` for zh-TW, `"#セキュリティ"` for ja. Simply setting this as `"#security"` may not be a good idea.

Comment (jhongyu) [reply kumascript/macros/HTTPSidebar.ejs]: I've updated the links to adapt for different languages

**Review by yin1999 — APPROVED** [review]
LGTM for 'zh-CN' and 'zh-TW'

**Comment by caugner** [comment]
@jhongyu Sorry for taking so long to get back to you.

Looking at that Security section on the Cookies page and seeing how small it is, I actually don't think it makes sense to have a dedicated link from the sidebar, so I would suggest to remove that link from the HTTPSidebar instead.

What do you think?

**Comment by jhongyu** [comment]
> @jhongyu Sorry for taking so long to get back to you.
> 
> Looking at that Security section on the Cookies page and seeing how small it is, I actually don't think it makes sense to have a dedicated link from the sidebar, so I would suggest to remove that link from the HTTPSidebar instead.
> 
> What do you think?

Sounds OK for me, I'll remove the link

**Review by caugner — APPROVED** [review]

**Pull request merged**